packages/vite-ts-config-paths.ts

```diff
@@ -0,0 +1,11 @@
+import viteTsConfigPathsModule from 'vite-tsconfig-paths';
+/**
+ * vite-tsconfig-paths does exports one way as CJS and another as ESM.
+ * This workaround allows us to use it in both contexts.
+ * @TODO: Remove this when vite-tsconfig-paths is fixed.
```

Comment:
Found a corresponding issue report in `vite-tsconfig-paths`.

- https://github.com/aleclarson/vite-tsconfig-paths/issues/75

Perhaps adding this link in the comment would help track progress if/when the issue is solved.

Reply:
Done in 98a56597, thank you!